app/helpers/confirmations_helper.rb

```diff
@@ -0,0 +1,2 @@
+module ConfirmationsHelper
+end
```

Comment:
I see that this helper method was not used anywhere. Why do we keep this file?

Reply:
@sku-wtag This was auto-generated. All the auto-generated files will be removed in future PRs. Thank for pointing.